app/helpers/patient_helper.rb

```diff
@@ -252,70 +252,81 @@ def normalize_and_get_state_name(name)
   end
 
   def timezone_for_state(name)
+    # Offsets are DST
     timezones = {
-      'alabama' => '-05:00',
-      'alaska' => '-08:00',
-      'americansamoa' => '-11:00',
-      'arizona' => '-07:00',
-      'arkansas' => '-05:00',
-      'california' => '-07:00',
-      'colorado' => '-06:00',
-      'connecticut' => '-04:00',
-      'delaware' => '-04:00',
-      'districtofcolumbia' => '-04:00',
-      'federatedstatesofmicronesia' => '+11:00',
-      'florida' => '-04:00',
-      'georgia' => '-04:00',
-      'guam' => '+10:00',
-      'hawaii' => '-10:00',
-      'idaho' => '-06:00',
-      'illinois' => '-05:00',
-      'indiana' => '-04:00',
-      'iowa' => '-05:00',
-      'kansas' => '-05:00',
-      'kentucky' => '-04:00',
-      'louisiana' => '-05:00',
-      'maine' => '-04:00',
-      'marshallislands' => '+12:00',
-      'maryland' => '-04:00',
-      'massachusetts' => '-04:00',
-      'michigan' => '-04:00',
-      'minnesota' => '-05:00',
-      'mississippi' => '-05:00',
-      'missouri' => '-05:00',
-      'montana' => '-06:00',
-      'nebraska' => '-05:00',
-      'nevada' => '-07:00',
-      'newhampshire' => '-04:00',
-      'newjersey' => '-04:00',
-      'newmexico' => '-06:00',
-      'newyork' => '-04:00',
-      'northcarolina' => '-04:00',
-      'northdakota' => '-05:00',
-      'northernmarianaislands' => '+10:00',
-      'ohio' => '-04:00',
-      'oklahoma' => '-05:00',
-      'oregon' => '-07:00',
-      'palau' => '+09:00',
-      'pennsylvania' => '-04:00',
-      'puertorico' => '-04:00',
-      'rhodeisland' => '-04:00',
-      'southcarolina' => '-04:00',
-      'southdakota' => '-05:00',
-      'tennessee' => '-05:00',
-      'texas' => '-05:00',
-      'utah' => '-06:00',
-      'vermont' => '-04:00',
-      'virginislands' => '-04:00',
-      'virginia' => '-04:00',
-      'washington' => '-07:00',
-      'westvirginia' => '-04:00',
-      'wisconsin' => '-05:00',
-      'wyoming' => '-06:00',
-      nil => '-04:00',
-      '' => '-04:00'
+      'alabama': { offset: -5, observes_dst: true },
```

Comment:
Will want to change this Hash back to being declared with the `=>` syntax or do a conversion of the key you use to access it. Right now whenever you try and grab a value in `timezones` it returns `nil` and so `-04:00` is always returned for the offset. Can update the keys to be symbols (not strings) to make it work with the colon syntax but that would require a conversion of the returned normalized_name on line 320: `normalize_name(name).to_s`

You can see current behavior by doing something like the following with default demo data:
```
Patient.all.collect {|p| "#{p.monitored_address_state}, #{p.address_timezone_offset}"}
```

Reply:
Might be good to write a quick test or two to just verify this function it's all good, if we can.

Reply:
Good catch, will convert these back to rocket style.